### ISSUE TITLE
Functionality - Changing Shift Multiple Selection

### ISSUE THREAD

**Issue by Jinsuke** [HP-Corel/AttendanceTracker#97]
Task
-
TL should be able to change a shift schedule in multiple selections of agents under his group.
@Ms. Evelyn, please analyze the flow and provide your feedback via Github (Create a new issue on this one).
@Jin, please create a mockup once Ms. Evelyn is done providing her feedback.

![image](https://cloud.githubusercontent.com/assets/10201846/11414478/ac81b516-9430-11e5-9173-aaf642f4a034.png)

**Comment by lyn2711** [comment]
Suggestion is to move the "More Actions" button beside the "Members" button and rename that to "Change Schedule".  Flow now would be:

1. Select All
2. Change Schedule
3. Show page where to input new shift and break schedules

**Comment by Jinsuke** [comment]
Mockup for Review
-
- Added Change Schedule feature on the right side of the Members Page. 

![groupschedmockup](https://cloud.githubusercontent.com/assets/10201846/11415730/72a48498-9441-11e5-8f8a-66235fa33f20.jpg)

**Comment by lyn2711** [comment]
This is okay. :)

**Comment by jin** [comment]
Hi guys, I think you have been tagging the wrong @Jin :)

**Comment by asjr4584** [comment]
Hi Jin,

Sorry for the wrong tagging. ☺

Thanks for letting us know.

Regards,

Alfredo A. Salvador, Jr.
Team Lead, Customer Solutions Rep V
HP ES Business Process Services

M + 639228901356
E alfredoa.salvadorjr@hpe.com<mailto:alfredoa.salvadorjr@hpe.com>
Hewlett Packard Enterprise Services
2F Two World Square
22 Upper McKinley Road, McKinley Town Center
Fort Bonifacio, Taguig City, 1634
Philippines

[cid:image002.png@01D0EBE8.E3043C30]

**Comment by asjr4584** [comment]
Deadline: December 30.

**Comment by Jinsuke** [comment]
Update
-
- UI is pushed to #97
![image](https://cloud.githubusercontent.com/assets/10201846/11713419/42580d4a-9f6e-11e5-90cd-f1ed65b16eb1.png)

**FUNCTIONALITY IS TO FOLLOW**

**Comment by asjr4584** [comment]
@Jinsuke @chrisperez @lyn2711 

Suggestions:
(1) Add the Select All button at the top of the Members list
(2) When no one is selected in the Members section, the Change Schedule is grayed out
